desktop/cmp/persistenceManager/PersistenceManager.ts

```diff
@@ -0,0 +1,214 @@
+import {div, fragment, hbox} from '@xh/hoist/cmp/layout';
+import {hoistCmp, HoistProps, PlainObject, uses} from '@xh/hoist/core';
+import {button} from '@xh/hoist/desktop/cmp/button';
+import {Icon} from '@xh/hoist/icon/Icon';
+import {menu, menuDivider, menuItem, popover} from '@xh/hoist/kit/blueprint';
+import {groupBy, keys, sortBy} from 'lodash';
+import {ReactNode} from 'react';
+import {manageDialog} from './impl/ManageDialog';
+import {saveDialog} from './impl/SaveDialog';
+import './PersistenceManager.scss';
+import {PersistenceManagerModel} from './PersistenceManagerModel';
+
+interface PersistenceManagerModelProps extends HoistProps {
```

Comment:
Believe our convention is to specify the model type as the generic argument to `HoistProps` (i.e. `interface PersistenceManagerModelProps extends HoistProps<PersistenceManagerModel>`). Then you don't need to specify the `model` type on line 14